### ISSUE TITLE
installing with ceph jewel

### ISSUE THREAD

**Issue by bengland2** [ceph/ceph-ansible#602]
When Barry Marson and I try to use Jewel release in Ceph gitbuilder, ceph-ansible fails at the point where it tries to start the monitor.  It clearly is thinking that it is still using sysvinit scripting, but Ceph has transitioned to systemd-native startup.  Also, I noticed that some of the tasks were determining which of systemd/upstart/sysvinit startup methods to use with tests roughly like "if is_infernalis_version" but shouldn't it really be "if version > hammer_version"?

TASK: [ceph-mon | start and add that the
          monitor service to the init sequence] *** 
failed: [gprfs025-10ge] => {"changed": false, "cmd":
          ["service", "ceph", "start", "mon"], "delta":
          "0:00:00.043230", "end": "2016-03-01 14:37:07.368086", "rc":
          1, "start": "2016-03-01 
14:37:07.324856", "warnings": ["Consider using service
          module rather than running service"]}
stderr: Redirecting to /bin/systemctl start  mon
          ceph.service
Failed to start mon.service: Unit mon.service failed to
          load: No such file or directory.
Warning: ceph.service changed on disk. Run 'systemctl
          daemon-reload' to reload units.
Job for ceph.service failed because the control process
          exited with error code. See "systemctl status ceph.service"
          and "journalctl -xe" for details.

**Comment by leseb** [comment]
`is_infernalis` already means > 0.94.
Usually the sysvinit script redirects to systemd, so this shouldn't be an issue.
Here it looks more like it can not find the unit file to start the daemon.

**Comment by bengland2** [comment]
I just pulled trunk after you said this was fixed and saw this:

TASK: [ceph-mon | start and add that the monitor service to the init sequence (for or after infernalis)] *** 
failed: [cephmon] => {"changed": false, "failed": true}
msg: Error when trying to enable ceph-mon@cephmon: rc=1 Failed to execute operation: No such file or directory.  

Looks like systemd unit file still not getting created.  This is just a single VM with all roles.

group_vars -- devices: /dev/vdb, journal_colocation: true, ceph_dev: true, ceph_dev_branch: jewel, pool_default_size: 1
OS = RHEL7.2.

**Comment by leseb** [comment]
argh let me try again on centos.

**Comment by leseb** [comment]
@bengland2 it seems to be working for me on:

```
[root@ceph-mon0 ~]# lsb_release -a
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: CentOS
Description:    CentOS Linux release 7.1.1503 (Core)
Release:        7.1.1503
Codename:       Core

[root@ceph-mon0 ~]# ceph -v
ceph version 10.0.4 (ea45099808051017fb05582555f126cba80567b8)
```

Weird...
